cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -406,13 +425,36 @@ def accelerator_experiments(self, key, value):
     """Translates accelerator_experiments field."""
     _extensions = self.get("extensions", {})
 
-    sub_a = clean_val("a", value, str)
-    sub_e = clean_val("e", value, str)
-    sub_p = clean_val("p", value, str)
-
     accelerators = _extensions.get("unit_accelerator", [])
     experiment = _extensions.get("unit_experiment", [])
     project = _extensions.get("unit_project", [])
+    institutions = _extensions.get("unit_institution", [])
+
+    val_a = clean_val("a", value, str)
+    val_e = clean_val("e", value, str)
+
+    def check_for_institution_in(val, value_to_check_for):
+        if value_to_check_for in val:
+            if value_to_check_for not in institutions:
+                institutions.append(value_to_check_for.upper())
+            val = val.replace(value_to_check_for, "")
+        return val.replace(" ", "")
```

Comment:
what if we have at some point an institution with two word name ? it will probably not happen for the migration but I wanted to make a point here that it would be safer to use `strip()` since in this way we replace all the spaces